jaxley_mech/solvers.py

```diff
@@ -121,9 +164,9 @@ def diffrax_implicit(
     dt: float,
     derivatives_func: Callable[..., jnp.ndarray],
     args: Tuple,
-    rtol: float = 1e-8,
-    atol: float = 1e-8,
-    max_iter: int = 4096,
+    term: ODETerm,
+    solver: ImplicitEuler,
```

Comment:
There's a misunderstanding here. It is probably due to the GitHub diff tool wrongly collapsed some lines... this line (`solver: ImplicitEuler`) is for your `diffrax_implicit`, I added this argument for future extensions, in case we want to use other solvers from diffrax.